test/run-integration-tests.yml

```diff
@@ -1,6 +1,7 @@
 parameters:
   dotnetSdkVersion: ''
   projectName: 'src/*.sln'
+  category: ''
```

Comment:
Do we really want to make this optional? Wouldn't it make sense to use `Integration` by default given we  most always use that or not?

Reply:
Thought we had that but if it's not true the that's fine for me!